### ISSUE TITLE
the mute/snooze mode/purpose post trip notification does not work

### ISSUE THREAD

**Issue by PatGendre** [e-mission/e-mission-docs#456]
Although the [post-trip-prompt.js code](https://github.com/e-mission/e-mission-phone/blob/master/www/js/tripconfirm/post-trip-prompt.js) still contains the MUTE/SNOOZE/CHOOSE actions that should enable to mute the post trip notifications, this doesn't seem to work any more since the incident notification was replaced by the mode/purpose confirm notification.

I don't understand why, and I checked our [fabmob code](https://github.com/fabmob/e-mission-phone-fabmob/blob/master/www/js/tripconfirm/post-trip-prompt.js), the file is the same as in e-mission-phone master.

**Comment by shankari** [comment]
@PatGendre can you clarify what you mean by "does not work"? Does it show different action buttons? Or does it show the same action buttons but clicking them has no effect (e.g. does not mute)?

**Comment by PatGendre** [comment]
@Shankari : the action buttons are not displayed.

**Comment by PatGendre** [comment]
@shankari FYI I changed my phone today and on the new phone (Nokia, android 9), the mute/snooze/choose action  button do appear, whereas it did not with my previous phone.

**Comment by shankari** [comment]
Interesting. your previous phone was a Huawei, right? Which version of android? I wonder if there are some versions of android which don't support action buttons in the notifications??!

**Comment by PatGendre** [comment]
yes it was a Huawei on Android 6.
I changed for the Huawei in May and the merge of the mode/purpose feature dates from may 7 so it can be that the incident notifications worked on my previous phone (a wiko on android 5, if I remember well) and I never tested them on the Huawei.
So it could due to a Huawei configuration, not to the Android version?

**Comment by shankari** [comment]
That is my guess given that the exact same code works on the Nokia and not on the Huawei. If you still have the Huawei around, I think that the a channel I created while at TRB in 2018 still has the old code. You can try it on both phones for another side by side comparison.

https://e-mission.eecs.berkeley.edu/#/client_setup?new_client=trb2018opendata&clear_usercache=true&clear_local_storage=true

If this is indeed a huawei issue, can we create a doc that lists model-specific issues that we have encountered? this will currently have only one entry, for the Huawei, which lists both the background restrictions and the lack of actions.

**Comment by PatGendre** [comment]
Hi @shankari  yes I still have the phone, I gave to my son who broke my older 2016 phone;-)
I could install emtriplog but when scanning but when I scan the QR code for the trb2018 UI, nothing happens. I wonder if this is depending on the Google account with which I installed the app?

**Comment by shankari** [comment]
Google doesn't even come into the picture until you actually start the onboarding process. oh wait, trb2018opendata apparently does not have an active commit. Let me see if I can find another old channel.

**Comment by shankari** [comment]
hm, try https://e-mission.eecs.berkeley.edu/#/client_setup?new_client=opentoall&clear_usercache=true&clear_local_storage=true
instead. This still uses the incident code 
https://github.com/e-mission/e-mission-phone/tree/opentoall/www/js
but slightly different buttons in the prompt (`WALK`, `BIKE`, `MORE`)

**Comment by PatGendre** [comment]
This time it worked:-)
But I get an error loading data while pushing/getting from server connection to https://emission.opensidewalks.com refused
I asked the question about Google because on G Play the store said that the app was already installed because I connected as me (not my son), but I understand it was a stupid question;-)

**Comment by shankari** [comment]
Yeah the opentoall project has shut down their server, although they may be starting a new one with the native app (https://github.com/e-mission/e-mission-docs/issues/410#issuecomment-558737048). But this should not affect the prompts at the end of the trip, which do not require any server intervention.

**Comment by PatGendre** [comment]
The post-trip notification works with the opentoall UI on the Huawei.

**Comment by shankari** [comment]
@PatGendre So to clarify, you can see the `WALK`, `BIKE` buttons from emTripLog on the Huawei but you cannot see the `MUTE`, `SNOOZE` buttons from the FabMob app?

**Comment by PatGendre** [comment]
@shankari yes, that's it. But I realise now that I did not see mute/snooze either for emtriplog opentoall UI.
